src/media/js/previews.js

```diff
@@ -12,28 +18,16 @@ define('previews',
 
     var slider_pool = [];
 
-    function adjustOffset() {
-        var trayOffset = 0;
-        // If the app shows notices we need to increase the container height.
-        // TODO: Clean this up better once we have an idea
-        // of whether we're happy with the desktop trays.
+    function adjustWidth() {
+        // Full width of window on desktop.
+        // Width of page on mobile.
+        var $previews = $('.previews');
         if (window.matchMedia('(min-width:1070px)').matches) {
-            if ($('.app-notices').children().length) {
-                trayOffset += 90;
-            }
-            if ($('#installed').css('display') !== 'none') {
-                trayOffset += 90;
-            }
-
-            $('.app-header.expanded > div')
-               .css('height', 778 + trayOffset + 'px')
-               .find('.previews')
-               .css('top', 225 + trayOffset + 'px');
+            var winWidth = z.win.width();
+            $previews.css('width', winWidth + 'px');
+            $previews.css('right', (winWidth - 1070) / 2 + 'px');
```

Comment:
I like to `Math.floor()` divisions but pixel fractions are valid anyway. Can leave it as is.